com.vrcfury.vrcfury/Editor/VF/Inspector/VRCFuryMaterialPropertySearchWindow.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEditor.Experimental.GraphView;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace VF.Inspector
+{
+    public class VRCFuryMaterialPropertySearchWindow : ScriptableObject, ISearchWindowProvider
```

Comment:
Doesn't need to happen in this pull, but it'd probably be nice to just make this generic and accept a list of strings / callback action when one is selected. That way we don't need to make a separate ISearchWindowProvider for each use case (since I'll probably be applying this to blendshapes).

Reply:
Yeah I thought about it since I wanted to make one for the blendshapes too, but since the amount of "window-specific" code is so small, I thought it would be ok for now until this is expanded into a more proper VRCF-specific API other actions could use

Reply:
Converted it to a generic SearchWindowProvider that can take an array of strings or a callback for generating search results, a callback for selecting and a userData `object` similar to other GraphView apis for any other data that might be useful